README.md

```diff
@@ -126,3 +126,75 @@ With that context in mind, once you've removed the manual things you may have ad
 ```
 docker run -it --env-file=kubefirst.env -v $PWD/gitops:/gitops -v $PWD/metaphor:/metaphor -v $PWD/scripts:/scripts -v $PWD/git:/git --entrypoint /scripts/nebulous/destroy.sh  nebulous:foo  
 ```
+
```

Comment:
what do you guys think to remove the previous README content? and continue working on the latest changes that are coming from GitLab?

Reply:
we might want to keep a link to our docs site but otherwise the content should be mostly out dated